test-end-to-end/android-maestro/common/accepting_course_join_request.yaml

```diff
@@ -0,0 +1,13 @@
+appId: com.toughra.ustadmobile
+---
+-  tapOn: "Courses"
+- tapOn:
+      id: "com.toughra.ustadmobile:id/item_clazzlist_clazz_cl"
+- tapOn: "MEMBERS"
+- tapOn:
+    id: "com.toughra.ustadmobile:id/accept_icon"
+- extendedWaitUntil:
+    notVisible:
+      id: "com.toughra.ustadmobile:id/accept_icon"
+    timeout: 10000
+- takeScreenshot: "results/StudentGotAdded"
```

Comment:
I think any screenshot that we take using the common yaml files needs to be a variable. If this common file is used by multiple different flows, then the screenshot will just get replaced.

Reply:
Ok Mike, got it. Thanks.